### DISCUSSION TITLE
How would I allow pistons to move globally

### DISCUSSION THREAD

**Discussion by piedart** [GriefPrevention/GriefPrevention#1091] [Q&A] [answered]
Hello, I've found the  'PistonMovement: CLAIMS_ONLY' Section, however was wondering what I would replace CLAIMS_ONLY wil to allow piston movement in all worlds.

**Comment by Jikoo** [comment, marked as answer]
PistonMovement options are documented [on the wiki](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#pistonmovement).
